src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/ProjectAnalysisPageIT.java

```diff
@@ -52,7 +52,7 @@ public void testGetProjectAnalysesOwner(){
 
 		assertTrue("This analysis submission should have job error info", analysisPage.hasJobErrorInfo());
 		analysisPage.displayInputFilesTab();
-		assertEquals("should be 2 files", 2, analysisPage.getNumberOfPairedEndInputFiles());
+		assertEquals("should be 1 pair of files", 1, analysisPage.getNumberOfPairedEndInputFiles());
```

Comment:
If the method `analysisPage.getNumberOfPairedEndInputFiles()` has changed to actually return the number of samples, then I think the name of the method should be changed as well.

Reply:
Updated in 2034bd9840128d567830e1850a7502f6a3db5c93